instrumentation/ratpack-1.4/library/src/main/java/io/opentelemetry/instrumentation/ratpack/RatpackHttpAttributesExtractor.java

```diff
@@ -63,7 +52,15 @@ protected String route(Request request) {
   @Override
   @Nullable
   protected String scheme(Request request) {
-    return null;
+    Context ratpackContext = request.get(Context.class);
+    if (ratpackContext == null) {
+      return null;
+    }
+    PublicAddress publicAddress = ratpackContext.get(PublicAddress.class);
+    if (publicAddress == null) {
+      return null;
+    }
+    return publicAddress.builder().build().getScheme();
```

Comment:
Can we use 
```suggestion
    return publicAddress.get().getScheme();
```
instead of building another copy?